restapi/src/main/java/io/stargate/web/resources/ColumnResource.java

```diff
@@ -45,19 +50,51 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+@Api(produces = MediaType.APPLICATION_JSON, consumes = MediaType.APPLICATION_JSON)
```

Comment:
I'm wondering if we should explicitly mention that these are all schema operations.

Reply:
adding